app/api/auth.py

```diff
@@ -269,3 +273,22 @@ def change_password():
         "name": user.fullname if user.fullname else None,
         "password-changed": True
     })
+
+@ticket_blueprint.route('/tickets/attendees/tickets/pdf/<string:order_identifier>/<string:filename>')
```

Comment:
what's the point of making a route if we have to give the filename in the URL? Are order identifiers not enough to get an order's tickets?

Reply:
I think the order identifier is enough to pull the tickets. I'll change it accordingly.